.github/CODEOWNERS

```diff
@@ -271,6 +271,7 @@ receiver/otlpjsonfilereceiver/                                      @open-teleme
 receiver/podmanreceiver/                                            @open-telemetry/collector-contrib-approvers @rogercoll
 receiver/postgresqlreceiver/                                        @open-telemetry/collector-contrib-approvers @djaglowski
 receiver/prometheusreceiver/                                        @open-telemetry/collector-contrib-approvers @Aneurysm9 @dashpole
+receiver/prometheusremotewritereceiver/                             @open-telemetry/collector-contrib-approvers @ArthurSens
```

Comment:
You should add the sponsor as a codeowner: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/CONTRIBUTING.md#adding-new-components

Reply:
@andrzej-stencel, I've added you here since you were the sponsor a few months ago. I'm not sure if you're still willing to be the sponsor though!

Let me know if you're not and I can look for another :)

Reply:
I'm going to be the sponsor going forward

Reply:
Thank you! I've replaced everything with your name :)